src/styles/main.scss

```diff
@@ -1,3 +1,205 @@
+    white-space: nowrap;
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: var(--card-main);
+    font-weight: 500;
+    cursor: pointer;
+
+    &:hover {
+      color: var(--card-secondary);
+    }
+
+    &:active,
+    &--active { /* keeping these for possible tests */
+      position: relative;
+      color: var(--card-secondary);
+    }
+
+    &:active::after,
+    &--active::after { /* keeping these for possible tests */
+      content: "";
+      border-radius: 5px;
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      height: 4px;
+      background-color: var(--card-secondary);
+    }
+  }
+
+  &__item + &__item {
+    margin-left: 20px;
+  }
+}
+
+.cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 944px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+.card {
+
+  width: 200px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid var(--card-border);
+  border-radius: 5px;
+  color: #060b35;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image-wrapper {
+    margin-bottom: 35px;
+    text-align: center;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+  }
+
+  &__review-count {
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    font-weight: 400;
+
+    &-amount {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+    }
+
+    &-desc {
+      margin: 0;
+    }
+  }
+
+  &__button-buy {
+    padding: 11px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #00acdc00;
+    border-radius: 5px;
+    background: var(--card-secondary);
+    color: #fff;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    &:hover {
+      border: 1px solid var(--card-secondary);
+      background: #fff;
+      color: var(--card-secondary);
+    }
+  }
+}
+
+.stars {
+
+  display: flex;
+
+  & > &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url("../images/star-active.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &--0 > &__star:nth-child(n+1),
```

Comment:
You can use @for loops for the stars selector as it mentioned in [Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

Reply:
I felt the other way was easier to understand, probably because not reading scss all too much. Just corrected.